pdr_backend/analytics/predictoor_dashboard/dash_components/callbacks.py

```diff
@@ -134,3 +137,40 @@ def update_feeds_table_on_search(search_value, feeds_data):
         # filter feeds by pair address
         filtered_data = filter_objects_by_field(feeds_data, "pair", search_value)
         return filtered_data
+
+    @app.callback(
+        Output("feeds_table", "selected_rows"),
+        [
+            Input("select-all-feeds_table", "n_clicks"),
+            Input("clear-all-feeds_table", "n_clicks"),
+        ],
+        [State("feeds_table", "data"), State("feeds_table", "selected_rows")],
+        prevent_initial_call=True,
+    )
+    def select_or_clear_all_feeds(_, __, rows, ___):
+        """
+        Select or clear all rows in the feeds table.
+        """
+
+        ctx = dash.callback_context
+        return select_or_clear_all_by_table(ctx, "feeds_table", rows)
+
+    @app.callback(
+        Output("predictoors_table", "selected_rows"),
+        [
+            Input("select-all-predictoors_table", "n_clicks"),
+            Input("clear-all-predictoors_table", "n_clicks"),
+        ],
+        [
+            State("predictoors_table", "data"),
+            State("predictoors_table", "selected_rows"),
```

Comment:
I don't think we need the selected rows inside the callbacks, from what I see they are not used